### PR TITLE
Minor Tweaks for PK borgs + recharger

### LINKED ISSUE #17677: Peacekeeping borgs not logging injections
Looking into a investigation involving a peacekeeping borg and it seems that the peacekeeping sedation drugs/hypospray is not logging when they are used.  I'm fairly certain most other injections are logged and mediborg stuff is logged too, so I think this is a bug that it isn't?

If not, close this.

### LINKED ISSUE #17918: ADD A COOLDOWN TO THE COOKIE MAKERS AHHHHHHRHGHHHHHH
http://i.imgur.com/fDMUBdF.png

### PR DESCRIPTION
**Pull request by silicons**
Cyborg rechargers now recharge a minimum of 200/tick, up from a default setting of 20/tick
PK borg cookie dispensers now have a 1 second cooldown
PK borg hug shocks has a 2 second cooldown up from 1
Cyborg hyposprays now log
PK borg harm alarm emagged will no longer stun you if you have bowman headset
PK borgs are the only cyborgs that can hugshock and hugcrush (I added hugs for it, medborgs, and standard borgs

fixes #17918 
fixes #17677

### CODE REVIEW

**Comment by kevinz000** [comment]
Logs work but won't log the name of the chemical. Shouldn't be that big of a problem since PK borg chemicals only are there to annoy the fuck out of/handicap people so if someone ahelps they're spamming shit at them atleast you know it's a bad chemical. But if anyone wants to 
TEaCH ME HOW TO GET CHEMICAL NAME TO LOG
that'd be nice

**Comment by Boredone** [comment]
Dunno if it'll work with the borghypo, but the normal hypospray (Like the CMO's) uses this:

			var/contained = english_list(injected)

			add_logs(user, M, "injected", src, "([contained])")

File: code/game/machinery/rechargestation.dm [hunk 1]
```diff
@@ -35,8 +35,7 @@
 	for(var/obj/item/weapon/stock_parts/manipulator/M in component_parts)
 		repairs += M.rating - 1
 	for(var/obj/item/weapon/stock_parts/cell/C in component_parts)
-		recharge_speed *= C.maxcharge / 10000
-
+		recharge_speed *= max((C.maxcharge / 10000), 1)
```
Comment (xxalpha) [inline code/game/machinery/rechargestation.dm]: This isn't any better for a fix, just restore the high capacity cell to cyborg rechargers.

Comment (kevinz000) [inline code/game/machinery/rechargestation.dm]: Got it

**Comment by kevinz000** [comment]
Annnd now? :^)

File: code/game/objects/items/weapons/RSF.dm [hunk 2]
```diff
@@ -179,4 +183,7 @@ RSF
 		R.cell.charge -= 100
 	else
 		matter--
-		desc = "A self recharging cookie fabricator. It currently holds [matter]/10 cookie-units."
\ No newline at end of file
+		desc = "A self recharging cookie fabricator. It currently holds [matter]/10 cookie-units."
+	cooldown = 1
+	spawn(cooldowndelay)
```
Comment (optimumtact) [inline code/game/objects/items/weapons/RSF.dm]: use cooldown = worldtime+cooldowndelay

and if (cooldown > worldtime)

instead of this spawn

Comment (kevinz000) [inline code/game/objects/items/weapons/RSF.dm]: k

**Pull request merged**